dev-haskell/ghc-exactprint/ghc-exactprint-1.8.0.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.5.1.9999
+#hackport: flags: -dev,-roundtrip
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="ExactPrint for GHC"
+HOMEPAGE="https://hackage.haskell.org/package/ghc-exactprint"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-haskell/data-default:=[profile?]
+	>=dev-haskell/free-4.12:=[profile?]
+	dev-haskell/ordered-containers:=[profile?]
+	>=dev-haskell/syb-0.5:=[profile?]
+	>=dev-lang/ghc-9.8.1:= <dev-lang/ghc-9.10
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-3.10.3.0
+	test? (
+		dev-haskell/cabal-syntax
+		dev-haskell/diff
+		dev-haskell/extra
+		>=dev-haskell/ghc-paths-0.1
```

Comment:
There are some changes in the dependencies that `hackport` detected (after some messing with the .cabal file so that it accepted it.)